arenadata/scripts/init_containers.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -eo pipefail
+
+project="$1"
+
+mkdir ssh_keys -p
+if [ ! -e "ssh_keys/id_rsa" ]
+then
+  ssh-keygen -P "" -f ssh_keys/id_rsa
+fi
+
+shift
+
+#up services specified in argument or all if service list is not specified
+docker-compose -p $project -f arenadata/docker-compose.yaml --env-file arenadata/.env up -d $@
+
+
+if [[ $# -eq 0 ]]; then
+  services=$(docker-compose -p $project -f arenadata/docker-compose.yaml config --services | tr '\n' ' ')
+else
+  services="$@"
```

Comment:
Why can't we just use the entire docker-compose.yaml file with the coordinator and three segments for resource group tests?

Reply:
If we don't care about running extra containers, then we can.

Reply:
I think this will be closer to reality.